src/buckets.ts

```diff
@@ -37,6 +38,10 @@ export class Buckets {
     }
   }
 
+  static fromUser(user: Users) {
```

Comment:
Just some syntactic sugar to make it easy to create a bucket api instance from an existing user instance

Reply:
This might actually be a bit much now that I think about it, just as easy to do `new Bucket(user.context)`

Reply:
Will revert